libraries/AP_RangeFinder/RangeFinder.h

```diff
@@ -190,12 +195,15 @@ class RangeFinder
 
     // return a 3D vector defining the position offset of the sensor in metres relative to the body frame origin
     const Vector3f &get_pos_offset(uint8_t instance) const {
-        return _pos_offset[instance];
+        return _RangeFinder_STATE(instance).pos_offset;
     }
     const Vector3f &get_pos_offset_orient(enum Rotation orientation) const;
 
-private:
+protected:
+
     RangeFinder_State state[RANGEFINDER_MAX_INSTANCES];
+
+private:
```

Comment:
Sorry, I forgot about this one earlier. Why is this change needed? I'm guessing it was also because of the *analog* detect method too?